cmd/license.go

```diff
@@ -0,0 +1,181 @@
+//
+// Copyright (c) 2021 SSH Communications Security Inc.
+//
+// All rights reserved.
+//
+
+package cmd
+
+import (
+	"github.com/SSHcom/privx-sdk-go/api/licensemanager"
+	"github.com/spf13/cobra"
+)
+
+type licenseOptions struct {
+	licenseKey string
+	optin      bool
+}
+
+func init() {
+	rootCmd.AddCommand(licenseListCmd())
+}
+
+//
+//
+func licenseListCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "license",
+		Short: "List and manage PrivX license",
+		Long:  `List and manage PrivX license keys`,
+		Example: `
+	privx-cli license [access flags]
+		`,
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return licenseList()
+		},
+	}
+
+	cmd.AddCommand(licenseSetCmd())
+	cmd.AddCommand(licenseRefreshCmd())
+	cmd.AddCommand(licenseStatisticsSetCmd())
+	cmd.AddCommand(licenseDeactivateCmd())
+
+	return cmd
+}
+
+func licenseList() error {
+	store := licensemanager.New(curl())
+
+	license, err := store.License()
+	if err != nil {
+		return err
+	}
+
+	return stdout(license)
+}
+
+//
+//
+func licenseSetCmd() *cobra.Command {
+	options := licenseOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "set",
+		Short: "Set new license",
+		Long:  `Set new license`,
+		Example: `
+	privx-cli license set [access flags] --key <LICENSE-KEY>
+		`,
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return licenseSet(options)
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.StringVar(&options.licenseKey, "key", "", "PrivX license key")
+
+	return cmd
+}
+
+func licenseSet(options licenseOptions) error {
+	api := licensemanager.New(curl())
+
+	err := api.SetLicense(options.licenseKey)
+	if err != nil {
+		return err
+	}
+
+	return err
```

Comment:
readability: return nil here or remove the error check above before returning err